libraries/AP_BattMonitor/AP_BattMonitor_Sum.cpp

```diff
@@ -73,6 +74,9 @@ AP_BattMonitor_Sum::read()
         if (!_mon.healthy(i)) {
             continue;
         }
+        if (voltage_count == 0 || _mon.voltage(i) < voltage_min) {
+            voltage_min = _mon.voltage(i);
+        }
         voltage_sum += _mon.voltage(i);
```

Comment:
Add the fetched voltage as a const float just to prevent getting it three times.